azurerm/provider_test.go

```diff
@@ -49,6 +52,21 @@ func testAccPreCheck(t *testing.T) {
 			t.Fatalf("`%s` must be set for acceptance tests!", variable)
 		}
 	}
+
+	// If "TF_PARL" is set to some non-empty value, all the parallel test cases
+	// will be run in parallel. It requires that the test cases are all
+	// independent ones without any shared resource before turn it on.
+	// example usage:
+	//  TF_ACC=1 TF_PARL=1 go test [TEST] [TESTARGS] -v -parallel=n
+	// To run specified cases in parallel, please refer to below PR:
+	// 	https://github.com/hashicorp/terraform/pull/16807
```

Comment:
I don't think we need these 4 lines, since they're included in the README?

Reply:
Different levels of details. README is based on Makefile while this is based on go test.